manifest-template.js

```diff
@@ -15,8 +15,10 @@ const manifest = {
 	"finish-args": [
 		"--share=ipc",
 		"--device=dri",
-		"--socket=x11",
+		"--socket=wayland",
+		"--socket=fallback-x11",
 		"--socket=pulseaudio",
+		"--socket=system-bus",
```

Comment:
This isn't allowed at all. You never should use this.

Reply:
you need ti use `--log-session-bus` to figure out the name you are missing.

Reply:
just to clarify this will break the build the linter WILL error out.